contracts/scripts/deployments/DeploySpaceFactory.s.sol

```diff
@@ -235,3 +243,50 @@ contract DeploySpaceFactory is DiamondDeployer {
   //   vm.stopBroadcast();
   // }
 }
+
+contract DeploySpaceFactoryV2 is DeploySpaceFactory {
```

Comment:
we should not be deploying 2 Diamond for such a small change, this will potentially deviate us from the upgrading pattern. I would propose either a legacy facet or just managing it in the new facet.